recipes/qemu-user-execve/conda_build_config.yaml

```diff
@@ -0,0 +1,20 @@
+emulated_targets: "aarch64 ppc64le s390x"
+# This is to avoid warning about not using {{stdlib('c')}} in the recipe
+emulated_c_stdlib:
+  - sysroot
+emulated_c_stdlib_version:
+  - "2.28"
+target_emulated_gcc:
+  - aarch64-conda-linux-gnu-gcc
+  - powerpc64le-conda-linux-gnu-gcc
+  - s390x-conda-linux-gnu-gcc
+target_emulated_platform:
+  - aarch64
+  - ppc64le
+  - s390x
+zip_keys:
+  - - target_emulated_platform
+    - target_emulated_gcc
+
+docker_image:
```

Comment:
Do you still need this? conda-forge switched to using an Alma9 image as the default.

Reply:
I thought I did, this recipe fails because it uses sysroot 2.17 that does not have the definition of MAP_FIXED_NOREPLACE in `mmap.h`. The value in the linux64.yaml seem to still be the cos7 (remember, I often don't really know what I am doing o.O)

Reply:
Ok, sorry, I forgot to remove the logging I used locally, so the error does not show in the log. next commit